source/OutfitterPanel.cpp

```diff
@@ -762,37 +1038,61 @@ void OutfitterPanel::DrawKey()
 	Color color[2] = {*GameData::Colors().Get("medium"), *GameData::Colors().Get("bright")};
 	const Sprite *box[2] = {SpriteSet::Get("ui/unchecked"), SpriteSet::Get("ui/checked")};
 
-	Point pos = Screen::BottomLeft() + Point(10., -VisibilityCheckboxesSize() - 20.);
-	Point off = Point(10., -.5 * font.Height());
+	Point pos = Screen::BottomLeft() + Point(10., -VisibilityCheckboxesSize() + checkboxSpacing / 2.);
+	const Point off{10., -.5 * font.Height()};
+	const Point checkboxSize{180., checkboxSpacing};
+	const Point checkboxOffset{80., 0.};
+
 	SpriteShader::Draw(box[showForSale], pos);
 	font.Draw("Show outfits for sale", pos + off, color[showForSale]);
-	AddZone(Rectangle(pos + Point(80., 0.), Point(180., 20.)), [this](){ ToggleForSale(); });
+	AddZone(Rectangle(pos + checkboxOffset, checkboxSize), [this](){ ToggleForSale(); });
+
+	pos.Y() += checkboxSpacing;
+	SpriteShader::Draw(box[showInstalled], pos);
+	// The text color will be "medium" when no ships are selected, regardless of checkmark state,
+	// indicating that the selection is invalid (invalid context).
+	font.Draw("Show installed outfits", pos + off, color[showInstalled && playerShip]);
```

Comment:
Perhaps, for consistency, change this to begin with "Show outfits":
```suggestion
	font.Draw("Show outfits installed", pos + off, color[showInstalled && playerShip]);
```